### PR TITLE
Prep 1.5.0 release

### PR DESCRIPTION
**Pull request by slim-bean**
Signed-off-by: Ed Welch <edward.welch@grafana.com>

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,253 @@
+## 1.5.0 (2020-05-20)
+
+It's been a busy month and a half since 1.4.0 was released, a lot has happened!
```
Comment (wardbekker) [inline CHANGELOG.md:3]: "and a lot of new improvements have been added to Loki since"

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,253 @@
...
+* [1920](https://github.com/grafana/loki/pull/1920) **alexanderGalushka**: feat: tms readiness check bypass implementation
+* [1894](https://github.com/grafana/loki/pull/1894) **cyriltovena**: Fixes possible panic in json pipeline stage.
+* [1865](https://github.com/grafana/loki/pull/1865) **adityacs**: Fix flaky promtail test
+* [1815](https://github.com/grafana/loki/pull/1815) **adityacs**: Log error message when source does not exist in extracted values
+* [1627](https://github.com/grafana/loki/pull/1627) **rfratto**: Proposal: Promtail Push API
+
+#### Docker Driver
+* [2076](https://github.com/grafana/loki/pull/2076) **cyriltovena**: Allows to pass inlined pipeline stages to the docker driver.
+* [2054](https://github.com/grafana/loki/pull/2054) **bkmit**: Docker driver: Allow to provision external pipeline files to plugin
+* [1906](https://github.com/grafana/loki/pull/1906) **cyriltovena**: Add no-file and keep-file log option for docker driver.
```
Comment (cyriltovena) [inline CHANGELOG.md:181]: May be we want to talk about this where we explained the configuration improvements. This is nice to know that doxker driver now clean logs

File: docs/operations/upgrade.md [hunk 3]
```diff
@@ -16,6 +20,53 @@ Cortex has done lot of cleanup in the configuration files, and you are strongly
...
+#### Test Your Config
+
+To see if your config needs to change, one way to quickly test is to download a 1.5.0 (or newer) binary from the [release page](https://github.com/grafana/loki/releases/tag/v1.5.0)
+
+Then run the binary providing your config file `./loki-linux-amd64 -config.file=myconfig.yaml`
+
+If there are configs which are no longer valid you will see errors immediately:
+
+```shell
+./loki -config.file=loki-local-config.yaml
```
Comment (owen-d) [inline docs/operations/upgrade.md:32]:
```suggestion
loki -config.file=loki-local-config.yaml
```

Comment (owen-d) [reply docs/operations/upgrade.md:32]: nit: they're likely not building from source/it probably won't be in local dir.

File: docs/operations/upgrade.md [hunk 4]
```diff
@@ -16,6 +20,53 @@ Cortex has done lot of cleanup in the configuration files, and you are strongly
...
+
+### Loki Docker Image User Changes
+
+To improve security concerns, in 1.5.0 the Docker container no longer runs the loki process as `root` and instead the process runs as user `loki` with UID `10001` and GID `10001`
+
+This may affect people in a couple ways:
+
+#### Loki Port
+
+If you are running Loki with a config that opens a port number above 1000 (which is the default, 3100 for HTTP and 9095 for GRPC) everything should work fine in regards to ports.
```
Comment (owen-d) [inline docs/operations/upgrade.md:55]: I think our default port was changed to 8080:
```
# HTTP server listen port
[http_listen_port: <int> | default = 80]
```

Comment (slim-bean) [reply docs/operations/upgrade.md:55]: was this default actually changed or is this just the weave default? (If it's the weave default we can't really change this in a clean way)

The config file that ships in the docker image sets it to 3100 so it's the "default" in docker

File: CHANGELOG.md (hunk 2 above)
Comment (slim-bean) [reply CHANGELOG.md:183]: done

**Pull request merged**